easybuild/easyconfigs/i/imkl/imkl-2021.1.1-iimpi-2020.12.eb

```diff
@@ -0,0 +1,14 @@
+name = 'imkl'
+version = '2021.1.1'
+
+homepage = 'https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html'
+description = "Intel oneAPI Math Kernel Library"
+
+toolchain = {'name': 'iimpi', 'version': '2020.12'}
+
+# see https://software.intel.com/content/www/us/en/develop/articles/oneapi-standalone-components.html
+source_urls = ['https://registrationcenter-download.intel.com/akdlm/irc_nas/17402/']
+sources = ['l_onemkl_p_%(version)s.52_offline.sh']
+checksums = ['818b6bd9a6c116f4578cda3151da0612ec9c3ce8b2c8a64730d625ce5b13cc0c']
+
```

Comment:
I noticed that the `postinstallcmds` block with the examples has been removed. Did you intend to move that to the easyblock?

Reply:
Yeah, but then I forgot about it. :D

That's done now in https://github.com/easybuilders/easybuild-easyblocks/pull/2430 (those changes will still work with existing `imkl` easyconfigs that still include the `postinstallcmds`, but we should clean those up).